### DISCUSSION TITLE
Leetcode question for JR Dev

### DISCUSSION THREAD

**Discussion by HirayaEs** [azl397985856/leetcode#576] [Q&A] [answered]
Is there a leetcode interview for fresh grad developer

**Comment by TianMeds** [comment, marked as answer]
I think based on my colleagues experience Jr dev sometimes doesn't have a leetcode interview its for seniors that has already gain experience

**Reply by hiluta233445678987654321** [reply]
I think the same that yu say that you think that you say i think.
